run/markdown-preview/renderer/main.js

```diff
@@ -19,11 +19,16 @@ const app = express();
 app.use(express.json());
 
 app.post('/', (req, res) => {
+  let markdown;
```

Comment:
Something like this would be more concise https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/master/run/pubsub/app.js#L15

Reply:
Done